pom.xml

```diff
@@ -1265,6 +1265,8 @@
     <osgiVersion>5.0.0</osgiVersion>
     <osgiEnterpriseVersion>5.0.0</osgiEnterpriseVersion>
     <slf4jVersion>1.7.7</slf4jVersion>
+    <snmp4jVersion>2.4.3</snmp4jVersion>
+    <snmp4jagentVersion>2.4.2</snmp4jagentVersion>
```

Comment:
snmp4j and snmp4j-agent are separate artifacts. At the moment, the versions above are the latest for the corresponding projects.